seungmi/kakao2022/주차요금계산.py

```diff
@@ -0,0 +1,38 @@
+import datetime as dt
+import math
+
+def cal_time(in_time,out_time):
+    time = str(dt.datetime.strptime(out_time,'%H:%M')-dt.datetime.strptime(in_time,'%H:%M'))
+    return int(time.split(":")[0])*60+int(time.split(":")[1])
+
+def solution(fees, records):
+    record_dic={}
+    fee_dic={}
+
+    #차량 별 사용시간 계산
+    #record_dic={"차량번호":[입차시간, 입/출차 여부,누적시간] }
+    for record in records:
+        car_info=record.split(" ")
```

Comment:
```suggestion
        time, number, type = record.split(" ")
```
인덱스 접근이 아닌 변수 이름을 명시해주는게 읽기에 편할 것 같아요!